app/views/tags/show.html.erb

```diff
@@ -42,7 +42,16 @@
     <% end %>
   </div>
   <div class="widget--body">
-    <%= raw(sanitize(@tag.wiki, scrubber: scrubber)) %>
+    <% if@tag.wiki.present? %>
+      <% if @tag.wiki.length < 600 %>
+      <%= raw(sanitize(@tag.wiki, scrubber: scrubber)) %>
```

Comment:
Content of if/else blocks should be indented one more level beyond the start of the block. This line and the HTML in the else block here need one more indent level.